src/app/(proper_react)/(redesign)/(authenticated)/admin/feature-flags/page.tsx

```diff
@@ -105,6 +105,14 @@ export default async function FeatureFlagPage() {
             user={session.user}
             monthlySubscriptionUrl={monthlySubscriptionUrl}
             yearlySubscriptionUrl={yearlySubscriptionUrl}
+            subscriptionBillingAmount={{
```

Comment:
A bit annoying now after the fact, so feel free to skip, but I wonder if we could do this in a separate function? That way, we have the (technically unsafe) type assertion in a single place, and we can add `import "server-only";` to catch situations where we're accidentally referencing a variable that doesn't exist on the client side.

Reply:
Good point — done in [526db2d](https://github.com/mozilla/blurts-server/pull/4083/commits/526db2db7c47a705d103f2acd49d2c8dae56868e).